From 9698fd904c1e971af6b9f2f3f19558fb2b4a8bcf Mon Sep 17 00:00:00 2001
From: Peter Ujfalusi <peter.ujfalusi@ti.com>
Date: Fri, 25 Jul 2014 15:07:48 +0300
Subject: [PATCH 1304/1587] ASoC: tlv320aic31xx: Add text in Kconfig so the
 codec can be used with simple-card

commit 96665e312940a0915e74a98cf75938a57d12c723 upstream

In order to show the option under 'CODEC drivers' we need to have text in
Kconfig.

Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
---
 sound/soc/codecs/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/codecs/Kconfig b/sound/soc/codecs/Kconfig
index 348e817..537084c 100644
--- a/sound/soc/codecs/Kconfig
+++ b/sound/soc/codecs/Kconfig
@@ -377,7 +377,7 @@ config SND_SOC_TLV320AIC26
 	depends on SPI
 
 config SND_SOC_TLV320AIC31XX
-        tristate
+	tristate "Texas Instruments TLV320AIC31xx CODECs"
 	depends on I2C
 	select REGMAP_I2C
 
-- 
1.7.5.4

